pkg/controller/fileintegrity/config_defaults.go

```diff
@@ -46,158 +46,24 @@ CONTENT_EX = sha512+ftype+p+u+g+n+acl+selinux+xattrs
 DATAONLY =  p+n+u+g+s+acl+selinux+xattrs+sha512
 
 /hostroot/boot/        CONTENT_EX
-/hostroot/opt/        CONTENT
 /hostroot/root/\..* PERMS
 /hostroot/root/   CONTENT_EX
+!/hostroot/root/.xauth*
 !/hostroot/usr/src/
 !/hostroot/usr/tmp/
 
 /hostroot/usr/    CONTENT_EX
 
-# trusted databases
-/hostroot/etc/hosts$      CONTENT_EX
-/hostroot/etc/host.conf$  CONTENT_EX
-/hostroot/etc/hostname$   CONTENT_EX
-/hostroot/etc/issue$      CONTENT_EX
-/hostroot/etc/issue.net$  CONTENT_EX
-/hostroot/etc/protocols$  CONTENT_EX
-/hostroot/etc/services$   CONTENT_EX
-/hostroot/etc/localtime$  CONTENT_EX
-/hostroot/etc/alternatives/ CONTENT_EX
-/hostroot/etc/sysconfig   CONTENT_EX
-/hostroot/etc/mime.types$ CONTENT_EX
-/hostroot/etc/terminfo/   CONTENT_EX
-/hostroot/etc/exports$    CONTENT_EX
-/hostroot/etc/fstab$      CONTENT_EX
-/hostroot/etc/passwd$     CONTENT_EX
-/hostroot/etc/group$      CONTENT_EX
-/hostroot/etc/gshadow$    CONTENT_EX
-/hostroot/etc/shadow$     CONTENT_EX
-/hostroot/etc/subgid$     CONTENT_EX
-/hostroot/etc/subuid$     CONTENT_EX
-/hostroot/etc/security/opasswd$ CONTENT_EX
-/hostroot/etc/skel/       CONTENT_EX
-/hostroot/etc/subuid$     CONTENT_EX
-/hostroot/etc/subgid$     CONTENT_EX
-/hostroot/etc/sssd/       CONTENT_EX
-/hostroot/etc/machine-id$ CONTENT_EX
-/hostroot/etc/system-release-cpe$ CONTENT_EX
-/hostroot/etc/shells$     CONTENT_EX
-/hostroot/etc/tmux.conf$  CONTENT_EX
-/hostroot/etc/xattr.conf$ CONTENT_EX
-
-# networking
-/hostroot/etc/hosts.allow$   CONTENT_EX
-/hostroot/etc/hosts.deny$    CONTENT_EX
+# OpenShift specific excludes
+!/hostroot/opt/
```

Comment:
What else is in /opt?

Reply:
Only the CNI plugins, but I suspect UPI systems could have stuff we don't control in there as well.

Reply:
OK, it makes sense to exclude this then.